lib/genome/importers/idg/idg_tsv_importer.rb

```diff
@@ -0,0 +1,27 @@
+module Genome
+  module Importers
+    module IDG
+      def self.source_info
+        {
+          base_url:           'https://druggablegenome.net/index/',
+          site_url:           'https://druggablegenome.net/index/',
+          citation:           'https://druggablegenome.net/IDGProteinList/',
```

Comment:
I would switch the base url and the citation url for to be more consistent with other importers.